### ISSUE TITLE
Inactive clr-tab doesn't have sufficient contrast (Accessibility WCAG 2.1)

### ISSUE THREAD

**Issue by PriteshGajjar008** [vmware/clarity#6264]
As per A11y guideline contrast ratio for inactive clr-tab is less.

For the inactivated tab,foreground color is #8C8C8C, background color is #FFF, color contrast is 3.36:1  
Can we change the foreground color of inactive tab to make the contrast ratio at least 4.5:1
![image](https://user-images.githubusercontent.com/20063009/129569761-e583003b-7f7c-42c0-b9f0-805092a0aa09.png)

Clarity version: v4

**Comment by Jinnie** [comment]
Fixed with 12.0.5 and 5.5.1
